statsmodels/genmod/cov_struct.py

```diff
@@ -938,6 +951,14 @@ class Equivalence(CovStruct):
     using this covariance structure, missing data should be removed
     before constructing the model.
 
+    If using `labels`, after a model is defined using the covariance
+    structure it is possible to remove a label pair from the second
+    level of the `pairs` dictionary to force the corresponding
+    covariance to be zero.
```

Comment:
Elements of the covariance matrices are 0 unless they are included in an
equivalence class.  When using labels, you don't have as much control over
what are the free parameters in the covariance structure.  For example, if
there are three label classes a, b and c, then you automatically get a/a,
b/b and c/c equivalence classes for variances, and a/b, a/c, and b/c
equivalence classes for covariances.  If for some reason you want to force
the b/c covariance to be zero, you would just delete the b/c label from all
the dicts.

On Wed, Jan 21, 2015 at 9:22 PM, Josef Perktold <notifications@github.com>
wrote:

> In statsmodels/genmod/cov_struct.py
> <https://github.com/statsmodels/statsmodels/pull/2188#discussion_r23351690>
> :
>
> > @@ -938,6 +951,14 @@ class Equivalence(CovStruct):
> >      using this covariance structure, missing data should be removed
> >      before constructing the model.
> >
> > +    If using `labels`, after a model is defined using the covariance
> > +    structure it is possible to remove a label pair from the second
> > +    level of the `pairs` dictionary to force the corresponding
> > +    covariance to be zero.
>
> This is not clear to me (based on reading just through the changes)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/statsmodels/statsmodels/pull/2188/files#r23351690>.
>